cogs/meme/meme.py

```diff
@@ -38,6 +38,20 @@ async def hug(self, ctx, user: discord.Member = None):
             return
 
         await ctx.send(emote.hug_right + f" **{self.sanitise(user.display_name)}**")
+    
+    @commands.command()
+    @commands.guild_only()
+    async def slap(self,ctx):
+        hugGifs = ["https://media1.tenor.com/images/612e257ab87f30568a9449998d978a22/tenor.gif?itemid=16057834","https://media1.tenor.com/images/153b2f1bfd3c595c920ce60f1553c5f7/tenor.gif?itemid=10936993","https://media1.tenor.com/images/9ea4fb41d066737c0e3f2d626c13f230/tenor.gif?itemid=7355956","https://media.tenor.com/images/091e0502e5fda1201ee76f5f26eea195/tenor.gif"]
+        randomGif = random.choice(hugGifs)
+        if len(ctx.message.mentions) == 1:
```

Comment:
These conditions do mostly the same. This can be done only once when you set `member` to `ctx.author` if it's None.